GeneralUtilities/inc/TwoDPoint.hh

```diff
@@ -0,0 +1,46 @@
+//
+//  Description of 2D points with covariance matrices
+//  This class works in SINGLE PRECISION for speed, so has limited accuracy
+//  Original author: Dave Brown (LBNL) 4/7/2023
+//
+#ifndef GeneralUtilities_TwoDPoint_hh
+#define GeneralUtilities_TwoDPoint_hh
+#include "Math/Vector3D.h"
+#include "Math/Vector2D.h"
+#include "Math/SVector.h"
+#include "Math/SMatrix.h"
+#include <iostream>
+namespace mu2e {
+  struct UVRes {
+    using VEC2 = ROOT::Math::XYVectorF; // spatial only 2D vector
+    VEC2 udir_; // u direction
+    float ures_, vres_; // resolution along u, v
+    float uvar() const { return ures_*ures_; }
+    float vvar() const { return vres_*vres_; }
+  };
+  class TwoDPoint {
+    public:
+      using VEC3 = ROOT::Math::XYZVectorF; // spatial only 3D vector
```

Comment:
I see that lots of code has using VEC3 - in a later PR does it make sense to definie it in one place and use it in many?  It's not clear to me where the right place is.  It might need to be in GeneralUtilities?